### DISCUSSION TITLE
Use libpq target_session_attrs=read-write to connect to postgresql pool

### DISCUSSION THREAD

**Discussion by nenominal** [netbox-community/netbox#6805] [General]
Hi,

we are using postgresql as db and I want to know if Netbox can be configured to connect to one pool using target_session to identify the master node.

Libpq has this ability and I'd like to know if it's implemented in connection settings of netbox and how to configure if it is.

Thanks.

**Comment by jeremystretch** [comment]
Looks like you can use Django's [`OPTIONS` parameter](https://docs.djangoproject.com/en/3.2/ref/settings/#std:setting-OPTIONS) within the `DATABASE` setting to pass this. I haven't tried it, but it probably goes like this:

```
DATABASE = {
    'NAME': 'netbox',
    ...
    'OPTIONS': {
        'target_session_attrs': 'read-write',
    },
}
```
